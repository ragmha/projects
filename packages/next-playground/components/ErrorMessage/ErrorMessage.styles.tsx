import styled from 'styled-components'

export const StyledErrorMessage = styled.aside`
  padding: 1.5em;
  font-size: 14px;
  color: white;
  background-color: red;
`

export default StyledErrorMessage
