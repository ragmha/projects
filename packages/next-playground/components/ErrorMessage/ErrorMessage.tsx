import { FC } from 'react'
import StyledErrorMessage from './ErrorMessage.styles'
import { ErrorMessageProps } from './ErrorMessage.types'

export const ErrorMessage: FC<ErrorMessageProps> = ({ message }) => (
  <StyledErrorMessage>{message}</StyledErrorMessage>
)

export default ErrorMessage
