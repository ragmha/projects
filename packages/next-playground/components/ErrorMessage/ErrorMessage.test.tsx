import ErrorMessage from './ErrorMessage'
import { render } from '@utils/test'

describe('ErrorMessage', () => {
  it('should match the snapshot', () => {
    const { container } = render(<ErrorMessage message="hello world!" />)
    expect(container).toMatchSnapshot()
  })
})
