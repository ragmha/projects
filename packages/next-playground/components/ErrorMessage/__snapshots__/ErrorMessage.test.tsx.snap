// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should match the snapshot 1`] = `
<div>
  <aside
    class="sc-AxjAm foezxt"
  >
    hello world!
  </aside>
</div>
`;
